src/python/pants/backend/docker/target_types.py

```diff
@@ -333,6 +334,55 @@ def option_values(self, **kwargs) -> Iterator[str]:
         yield from cast("tuple[str]", self.value)
 
 
+class DockerBuildUnaryOptionFieldMixin(Field):
+    """Inherit this mixin class to provide unary options (i.e. option in the form of `--flag=value`)
+    to `docker build`."""
+
+    docker_build_option: ClassVar[str]
+
+    @final
+    def options(self) -> Iterator[str]:
+        yield f"{self.docker_build_option}={self.value}"
```

Comment:
Does the docker CLI take `True`/`False` capitalized? Should we be lower-casing here?

Reply:
Yes. I checked with Docker CLI on my end and it does take capitalized `True` / `False`